dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracleAppConfig.scala

```diff
@@ -73,10 +73,14 @@ case class DLCOracleAppConfig(
   }
 
   lazy val aesPasswordOpt: Option[AesPassword] = {
```

Comment:
Is there a reason we don't just create a `KeyManagerAppConfig` and make `DLCOracleAppConfig` have-a `KeyManagerAppConfig`?

Same thing applies to the `WalletAppConfig`

Reply:
I guess we could, may be better for a future PR so we could integrate all the key manager stuff to use it as well

Reply:
#2255